opencti-platform/opencti-front/src/private/components/common/files/FileLine.tsx

```diff
@@ -285,7 +284,7 @@ const FileLineComponent: FunctionComponent<FileLineComponentProps> = ({
             }
           />
         </Tooltip>
-        <ListItemSecondaryAction>
+        <div onClick={(e) => e.stopPropagation()} style={{ flex: 'none' }}>
```

Comment:
Neither am I.
The onClick handler might not be necessary. @lndrtrbn ?
I have noticed an ellipse in the style.

Reply:
`ListItemSecondaryAction` => MUI component to add one and only one icon button at the end of the row. So not our case, that's why text is overlapping the buttons.

Instead just use a div, the `stopPropagation` is needed to avoid calling the onClick callback when clicking on a line (which redirect to a new page)

Reply:
Seem with @Goumies we will rollback the change and update the text to truncate it before it reaches the buttons

@lndrtrbn I completely align with your vision, we are incorrectly using MUI components for list/lines & co, but this is a larger concern that we will need to tackle everywhere.
In fact it's mostly that we should not have been using ListItem components

Reply:
That will be a mess to compute where you need to truncate the text as you don't really know how much buttons you have and what is the width of the row.

To achieve this you need to compute row width, ListItemSecondaryAction width, icon width, set the text width to (row - icon - ListItemSecondaryAction) and truncate using CSS

Reply:
My idea is to truncate text with a drastic limit which will be enough for the smallest available width.
A tooltip will reveal the entire text.